examples/connext_dds/asynchronous_publication/c++98/application.h

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
If we're changing to the new templates. I think we don't need to check in application.h, because it'll be generated by rtiddsgen, similarly to the rest of cxx and h files.

Reply:
Actually, it is necessary. The CMake build infrastructure does not generate the application.h files, and some application.h has been modified because there are some examples with different or extra arguments.